Makefile

```diff
@@ -68,8 +68,10 @@ luakit.1: luakit.1.in
 luakit.1.gz: luakit.1
 	@gzip -c $< > $@
 
-apidoc: luakit luakit.so
-	rm -rf doc/apidocs
+clean-apidoc:
+	-rm -rf doc/apidocs
+
+apidoc: clean-apidoc luakit luakit.so
```

Comment:
I can't see how this is related to fixing #363. Ditto with the other changes except `+install: all`.

If this fixes a different problem, please open a separate issue/PR; thanks!